lapce-core/Cargo.toml

```diff
@@ -22,6 +23,8 @@ lapce-rpc = { path = "../lapce-rpc" }
 
 tree-sitter = "0.20.7"
 
+# plaintext grammar, not optional
+tree-sitter-plaintext = { git = "https://github.com/panekj/tree-sitter-plaintext", rev = "38c37a098fea0eec26c6b0aeaa9feaec8bbe93fe" }
```

Comment:
This seems to be working just fine, but can you clarify why this needs to be a tree-sitter grammar? Is it just to make interop with the rest of the "real" tree-sitter langs easier? Does this mean that all of the `find_tag()`, `find_tag_in_siblings()`, etc methods in `Syntax` will start working for "text" files, or will those still end up using `WordCursor`?